package.json

```diff
@@ -3,31 +3,41 @@
   "version": "0.1.0",
   "private": true,
   "proxy": "http://localhost:8080",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "lint:fix": "eslint --fix --ext .js,.jsx,.ts,.tsx --ignore-pattern '/node_modules/' ./src"
+  },
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@types/cytoscape": "^3.14.7",
     "@types/jest": "^24.0.0",
+    "@types/luxon": "^1.25.0",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-cytoscapejs": "^1.1.0",
     "@types/react-dom": "^16.9.0",
+    "clsx": "^1.1.1",
     "cytoscape": "3.15.0",
+    "luxon": "^1.25.0",
```

Comment:
This package is used in one location, and does not really bring any benefits to the current usage.

Reply:
Fair for now. But I also wanted to use it to compute the expiration time on the portals instead of hours/minutes, give the user an actual local or UTC time of collapse.